dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_42__make_shortname_in_trackedentitytype_unique.sql

```diff
@@ -0,0 +1,2 @@
+alter table trackedentitytype drop constraint if exists trackedentitytype_shortname_unique;
```

Comment:
Maybe this is not a written convention yet, but we have tried to follow 

uk_tablename_columnname format in the past. So this key name should be

`uk_trackedentitytype_shortname`

Reply:
thanks, updated now.